ts/src/renderer/phaser/classes/PhaserUnit.ts

```diff
@@ -175,6 +189,17 @@ class PhaserUnit extends PhaserAnimatedEntity {
 		}
 	}
 
+	private scaleElements (height): void {
+		const defaultZoom = ige.game.data.settings.camera?.zoom?.default || 1000;
```

Comment:
@Grenagar why are we using zoom from the game stats here instead of a value that already exists on the main camera object?